.github/ISSUE_TEMPLATE/questions-help-support.md

```diff
@@ -0,0 +1,16 @@
+---
+name: "❓Questions/Help/Support"
+about: Do you need support? We have resources.
+
+---
+
+## ❓ Questions and Help
+
+### Please note that this issue tracker is not a help form and this issue will be closed.
+
+We have a set of [listed resources available on the website](https://pytorch.org/resources). Here is a set of support channels, in order of where you should begin looking for help:
+
+- [Stack Overflow](https://stackoverflow.com/questions/tagged/pytorch)
```

Comment:
We should remove StackOverflow, Twitter and Slack as channels for support.

The single and only channel PyTorch emphasizes for support is [Discussion Forum](https://discuss.pytorch.org/)